libraries/postgres/src/Database.js

```diff
@@ -642,6 +649,57 @@ class Database {
       this.monitor.log.dbPoolCounts(fields);
     }
   }
+
+  /**
+   * Takes any bytes value (you must ensure this as the caller, e.g. `Buffer.from()`)
+   * and returns an encrypted value for storing in the db with the current crypto key
+   *
+   * This currently only supports lib-entities shaped data but can be extended to support
+   * other formats if needed. The "property name" is hardcoded to `val` since we only
+   * have one property.
+   */
+  encrypt({value}) {
+    const {id, key} = this.keyring.currentCryptoKey('aes-256');
+
+    const iv = crypto.randomBytes(16);
+    const cipher = crypto.createCipheriv('aes-256-cbc', key, iv);
+    const c1 = cipher.update(value);
+    const c2 = cipher.final();
```

Comment:
@ajvb compare to https://github.com/taskcluster/taskcluster/blob/0111d91b6a4ccefb6594444a7d30a7820711f3ff/libraries/entities/src/entitytypes.js#L517-L536